ts/src/pro/poloniex.ts

```diff
@@ -0,0 +1,970 @@
+        /**
+         * @method
+         * @name poloniex#watchOrders
+         * @description watches information on multiple orders made by the user
+         * @see https://docs.poloniex.com/#authenticated-channels-market-data-orders
+         * @param {string|undefined} symbol unified market symbol of the market orders were made in
+         * @param {int|undefined} since not used by poloniex watchOrders
+         * @param {int|undefined} limit not used by poloniex watchOrders
+         * @param {object} params extra parameters specific to the poloniex api endpoint
+         * @returns {[object]} a list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
+         */
+        await this.loadMarkets ();
+        const name = 'orders';
+        await this.authenticate ();
+        const symbols = (symbol === undefined) ? undefined : [ symbol ];
+        const orders = await this.subscribe (name, true, symbols, params);
+        if (this.newUpdates) {
+            limit = orders.getLimit (symbol, limit);
+        }
+        return this.filterBySinceLimit (orders, since, limit, 'timestamp', true);
+    }
+
+    async watchBalance (params = {}) {
+        /**
+         * @method
+         * @name poloniex#watchOrders
+         * @description watches information on multiple orders made by the user
+         * @see https://docs.poloniex.com/#authenticated-channels-market-data-balances
+         * @param {string|undefined} symbol not used by poloniex watchBalance
+         * @param {int|undefined} since not used by poloniex watchBalance
+         * @param {int|undefined} limit not used by poloniex watchBalance
+         * @param {object} params extra parameters specific to the poloniex api endpoint
+         * @returns {[object]} a list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
+         */
+        await this.loadMarkets ();
+        const name = 'balances';
+        await this.authenticate ();
+        return await this.subscribe (name, true, undefined, params);
+    }
+
+    parseWsOHLCV (ohlcv, market = undefined) {
+        //
+        //    {
+        //        symbol: 'BTC_USDT',
+        //        amount: '840.7240416',
+        //        high: '24832.35',
+        //        quantity: '0.033856',
+        //        tradeCount: 1,
+        //        low: '24832.35',
+        //        closeTime: 1676942519999,
+        //        startTime: 1676942460000,
+        //        close: '24832.35',
+        //        open: '24832.35',
+        //        ts: 1676942492072
+        //    }
+        //
+        return [
+            this.safeInteger (ohlcv, 'startTime'),
+            this.safeNumber (ohlcv, 'open'),
+            this.safeNumber (ohlcv, 'high'),
+            this.safeNumber (ohlcv, 'low'),
+            this.safeNumber (ohlcv, 'close'),
+            this.safeNumber (ohlcv, 'amount'),
```

Comment:
You want to use the base volume
```suggestion
            this.safeNumber (ohlcv, 'quantity'),
```